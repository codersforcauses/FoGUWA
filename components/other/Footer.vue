<template>
  <v-footer class="px-4">
    <div class="mx-1">Made with &#10084; by Coders for Causes</div>
  </v-footer>
</template>

<script>
export default {}
</script>
