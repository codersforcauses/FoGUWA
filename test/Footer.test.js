import Vuetify from 'vuetify'
import { mount, createLocalVue } from '@vue/test-utils'
import Footer from '@/components/other/Footer.vue'

const localVue = createLocalVue()

describe('Footer.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  test('mount component', () => {
    const wrapper = mount(Footer, {
      localVue,
      vuetify
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
