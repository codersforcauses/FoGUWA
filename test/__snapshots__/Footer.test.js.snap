// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue mount component 1`] = `
"<v-footer class=\\"px-4\\">
  Made with ‚ù§ by Coders for Causes
</v-footer>"
`;
